src/components/Tokens/TokenTable/TimeSelector.tsx

```diff
@@ -111,7 +111,7 @@ export default function TimeSelector() {
 
   return (
     <StyledMenu ref={node}>
-      <FilterOption onClick={toggleMenu} aria-label="timeSelector" active={open}>
+      <FilterOption onClick={toggleMenu} aria-label="timeSelector" active={open} data-testid="time-selector">
```

Comment:
Use the aria-label instead of a testid?

Reply:
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements 
^ suggests using testid to avoid changes (eg: aria label could change)